docs/examples/c-code/co_simulation.c

```diff
@@ -38,27 +38,20 @@ int main(int argc, char* argv[]) {
     fmi3CallbackFunctions callbacks = { NULL };
     fmi3Instance s1, s2;
 
-    fmi3CoSimulationConfiguration coSimConfiguration;
     printf("Running CoSimulation example... ");
 
 // tag::CoSimulation[]
 ////////////////////////////
 // Initialization sub-phase
 
 // set callback functions
-callbacks.logMessage          = cb_logMessage;
-callbacks.allocateMemory      = cb_allocateMemory;
-callbacks.freeMemory          = cb_freeMemory;
-callbacks.instanceEnvironment = NULL;
-
-coSimConfiguration.intermediateVariableGetRequired = fmi3False;
```

Comment:
shouldn't we show in the co-sim example how to use the coSimConfiguration structure instead of just using a NULL pointer?

Reply:
To save time and development resources my suggestion would be to contentrate the development of an example here https://github.com/CATIA-Systems/Test-FMUs/tree/feature/clocks-and-hybrid-cs and reuse the results as an example (include only relevant parts based on tagging) in the document.